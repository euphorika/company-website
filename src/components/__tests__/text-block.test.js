import React from "react"
import renderer from "react-test-renderer"

import TextBlock from "../text-block"

describe("TextBlock", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<TextBlock title="Awesome Headline">Awesome Content</TextBlock>)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
