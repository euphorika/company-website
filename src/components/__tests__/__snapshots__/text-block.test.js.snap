// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextBlock renders correctly 1`] = `
<section
  className="textBlockContainer"
>
  <h1
    dangerouslySetInnerHTML={
      Object {
        "__html": "Awesome Headline",
      }
    }
  />
  <div
    className="body"
  >
    Awesome Content
  </div>
</section>
`;
